magento/inventory
# Msi 2631 Source selector disappear in store pickup orders placement from admin panel

### Description (*)
Now after the shipping method reset (In case 'Save in address book' for country fields are changed)
In-Store Pickup shipping method is selected if it previously was.

### Fixed Issues (if relevant)
1. https://github.com/magento/inventory/issues/2631 Source selector disappear in store pickup orders placement from admin panel

### Manual testing scenarios (*)
1. Place order with your product in admin area
2. Select 'In-Store Pickup Delivery' as shipping method 
3. Verify that drop-down list for 'Pickup Location Source' selection appear 
4. Uncheck box 'Save in address book' for your billing address
5. Make Sure 'In-Store Pickup Delivery' is still selected and the sources list is visible

### Contribution checklist (*)
